lib/msal-core/src/utils/ErrorUtils.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AuthCache } from "../cache/AuthCache";
+import { AuthError } from "../error/AuthError";
+/**
+ * @hidden
+ */
+export class ErrorUtils {
+
+    static throwErrorAndClearTempCache(storage: AuthCache, error: AuthError): AuthError {
```

Comment:
Same comment as before - this should either be a part of the AuthError class/file, or it we should have a class that does control flow handling, including functions from UserAgentApplication which handle errors and responses. Creating a utils specifically for errors doesn't need to be done in my opinion.

Reply:
Addressing this, made it a helper function in UAA.